src/content/docs/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -128,6 +129,26 @@ Relationships between a service and host entity require that the service
 includes the `host.id` resource attribute and that it matches the `host.id` of
 the host it is running on.
 
+### Service to AWS resources 
+
+#### Service to AWS Elasticache Memcached Cluster 
+
+Relationships between a service and AWS Memcached Cluster entity requires the following:
+
+Service should generate the following span attributes:
+
+* `db.system`: `memcached`
+* `net.peer.name`: AWS connection endpoint of the memcached clusterr ending with `cache.amazonaws.com`
```

Comment:
```suggestion
* `net.peer.name`: AWS connection endpoint of the memcached cluster ending with `cache.amazonaws.com`
```

Also, where does "ending with `cache.amazonaws.com`" show up in the entity synthesis rules?